tensorflow/compiler/xla/python/pytree.h

```diff
@@ -144,8 +144,7 @@ class PyTreeDef {
   std::vector<std::unique_ptr<PyTreeDef>> Children() const;
 
   // Maps a function over a PyTree structure, applying f_leaf to each leaf, and
-  // f_node to each container node.
-  // TODO(phawkins): use flattening everywhere instead and delete this method.
```

Comment:
I'm wondering about removing this, as this now provides functionality that flattening doesn't?